lib/search_test.go

```diff
@@ -28,23 +28,30 @@ func TestSearch(t *testing.T) {
 		t.Fatal(err.Error())
 	}
 
-	// Case 0 - request with expected result
-	i := 0
-	p := &SearchParams{"cities", 0, 100}
-	numResults := 3
-	errString := ""
+	cases := []struct {
+		description string
+		params      *SearchParams
+		numResults  int
+		err         string
+	}{
+		{"search datasets - 'cities'", &SearchParams{"cities", 0, 100}, 3, ""},
+		// {"search datasets - 'cities'", &SearchParams{"cities", 0, 2}, 2, ""},
```

Comment:
either remove or enable, I'm totally fine with just removing for now. This PR is already pretty solid 😄